python/dgl/network.py

```diff
@@ -318,5 +327,4 @@ def _clear_kv_msg(garbage_msg):
     for msg in garbage_msg:
         if msg.c_ptr is not None:
             _CAPI_DeleteKVMsg(msg.c_ptr)
-    garbage_msg = []
-        
\ No newline at end of file
+    garbage_msg = []
```

Comment:
Is this necessary for a input parameter?

Reply:
I think garbage_msg is necessary for all the cases.